src/lib/components/WebvizContainerPlaceholder.react.js

```diff
@@ -0,0 +1,117 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import htmlToImage from 'html-to-image';
+
+import { faFileCsv,
+         faAddressCard,
+         faQuestionCircle,
+         faCameraRetro,
+         faExpand } from '@fortawesome/free-solid-svg-icons'
+
+import WebvizToolbarButton from '../private-components/webviz-container-placeholder-resources/WebvizToolbarButton'
+import WebvizContentOverlay from '../private-components/webviz-container-placeholder-resources/WebvizContentOverlay'
+import download_file from '../private-components/webviz-container-placeholder-resources/download_file'
+
+import './webviz_container_component.css'
+
+/**
+ * WebvizContainerPlaceholder is a fundamental webviz dash component.
+ * It takes a property, `label`, and displays it.
+ * It renders an input with the property `value` which is editable by the user.
+ */
+export default class WebvizContainerPlaceholder extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            expanded: false,
+            showOverlay: false
+        }
+    }
+
+    componentDidUpdate(prevProps) {
+        if (this.props.csv_string !== '') {
```

Comment:
I think should also check against prevProps to prevent infinite loops?
```
if (this.props.csv_string !== this.prevProps.csv_string) {
    if (this.props.csv_string !== '') {
    ...
```